app/components/candidate_interface/degree_required_component.rb

```diff
@@ -0,0 +1,75 @@
+class CandidateInterface::DegreeRequiredComponent < ViewComponent::Base
+  CANDIDATE_GRADES = {
+    'Upper second-class honours (2:1)' => 4,
+    'Lower second-class honours (2:2)' => 3,
+    'Third-class honours' => 2,
+    'Pass' => 1,
+  }.freeze
+
+  COURSE_REQUIRED_GRADES = {
+    'two_one' => 4,
+    'two_two' => 3,
+    'third_class' => 2,
+  }.freeze
+
+  COURSE_REQUIRED_GRADE_TEXT = {
+    'two_one' => '2:1 degree or higher (or equivalent)',
+    'two_two' => '2:2 degree or higher (or equivalent)',
+    'third_class' => 'Third-class degree or higher (or equivalent)',
+    'not_required' => 'Any degree grade',
+  }.freeze
+
+  DEGREE_GRADE_INSUFFICIENT_TEXT = {
+    'Lower second-class honours (2:2)' => 'a 2:2 degree.',
+    'Third-class honours' => 'a Third class degree.',
+    'Pass' => 'an Ordinary degree (pass).',
```

Comment:
@B-Bonner should w change this to:

```suggestion
    'Pass' => 'a Pass (Ordinary degree).',
```

Reply:
Well, well I looked at example degree certificates online, its written as and 'Ordinary degree'. I put pass in brackets to make it easier for users who aren't familiar with that term.